Another-application/Build-Container-Images.ps1

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
maybe I missed this discussion, could you please reiterate why we need another application?

Reply:
The bookstore application was not working, we were getting the asp net development mode error. So keeping that in mind, I got another application ready just in case.

Reply:
if that's the case and the original app is not working, we should also investigate into modifying/removing the bookstore application.

Reply:
The bookstore application is fixed and working.

Reply:
If this is fixed and working, do we still need the second application?

Reply:
Discussed in person with the team.